lms/djangoapps/discussion/views.py

```diff
@@ -671,9 +691,20 @@ def render_to_fragment(self, request, course_id=None, discussion_id=None, thread
 
             fragment = Fragment(html)
             self.add_fragment_resource_urls(fragment)
-            fragment.add_javascript(inline_js)
+
+            if request.GET.get('format', 'html') == 'json':
+                fragment.add_javascript_url(staticfiles_storage.url('common/js/vendor/require.js'))
+                fragment.add_javascript_url(staticfiles_storage.url('lms/js/require-config.js.js'))
+
             if not settings.REQUIRE_DEBUG:
-                fragment.add_javascript_url(staticfiles_storage.url('discussion/js/discussion_board_factory.js'))
+                fragment.add_javascript_url(
+                    staticfiles_storage.url(
+                        staticfiles_storage.url('discussion/js/discussion_board_factory.js')
```

Comment:
@bogdanl Why double `staticfiles_storage.url`?

Reply:
Oh sorry about that, rushed commit – thanks for pointing it out.